lib/tools/searchWeb/getSearchWebTool.ts

```diff
@@ -1,33 +1,65 @@
 import { z } from "zod";
 import { tool } from "ai";
-import performChatCompletion from "@/lib/perplexity/performChatCompletion";
```

Comment:
YAGNI principle
- actual: It looks like you're no longer using the `chat completion` API from Perplexity. Why are we leaving unused code in our codebase?
- required: delete the `@/lib/perplexity/performChatCompletion` and any unused libs resulting from your changes.